cql3/query_options.cc

```diff
@@ -161,23 +161,6 @@ query_options::query_options(std::vector<cql3::raw_value> values)
           db::consistency_level::ONE, infinite_timeout_config, std::move(values))
 {}
 
-bytes_view query_options::linearize(fragmented_temporary_buffer::view view) const
```

Comment:
IIRC the goal was to use this to store inputs from transport that happened to span several packets. Let me check.

Reply:
Introduced:

```
commit 8f4cb36ef2f7bf81e2054d3f20c8e2107ed15ebe
Author: Paweł Dziepak <pdziepak@scylladb.com>
Date:   Wed Jul 11 13:53:03 2018 +0100

    cql3 query_options: add linearize()
    
    Some code in the CQL3 layer requires bytes_view and it is fairly
    reasonable to assume that it won't deal with large buffers (e.g.
    statement restrictions). query_options already has make_temporary()
    which takes ownership of a cql3::raw_value so that the rest of the code
    can use cql3::raw_value_view. This patch adds similar linearize()
    function which, if necessary, linearises a cql3::raw_value_view and
    returns a bytes_view with lifetime tied to the life or query_options.
```

Reply:
No, can't find any user.

Reply:
(in history)